BlueprintUI/Sources/BlueprintView/ElementPreview.swift

```diff
@@ -191,34 +191,28 @@ extension ElementPreview {
             for element: Element
         ) -> AnyView {
 
-            let formattedName: String = {
-                if name.isEmpty == false {
-                    return " – " + name
-                } else {
-                    return ""
-                }
-            }()
+            let formattedName: String? = name.isEmpty ? nil : name
```

Comment:
I'm not actually sure, and fine keeping this in, but should we just allow empty string here? We could document that it maps to `.previewDisplayName()` so consumers know it matches that behavior.
What happens if you do provide empty string?

Reply:
if you provide an empty string here it defaults to the SwiftUI default. I _didn't_ want to change it to trinary (nil, empty, named), because that'd break the interface.

If you pass in an empty string it should be the same as it was before

Reply:
if you pass in an empty string you get the SwiftUI default, if you pass in a name it shows the name. I could have made this trinary (`String?`) and have `nil` give you the swiftUI default, empty string keep the same behavior ("size that fits" etc.), and a non-empty string override the preview name.

Reply:
Eh, I kinda like the idea of having the `name` match the type of `previewDisplayName` but probably not worth the churn. Fine keeping as is!